api/types/types.proto

```diff
@@ -1587,6 +1587,11 @@ message RoleOptions {
     // CertExtensions specifies the key/values
     repeated CertExtension CertExtensions = 17
         [ (gogoproto.jsontag) = "cert_extensions,omitempty" ];
+
+    // MaxKubernetesConnections defines the maximum number of concurrent
+    // Kubernetes sessions a user may hold.
+    int64 MaxKubernetesConnections = 18
```

Comment:
It's more of an question rather than an issue, but do we have any guidelines for using specific scalar value types inside protobuf types? I'm asking just because an int64 seems like an overkill for number of connections, but at the same time I've seen that in other protobuf types we also tend to use int64, e.g: https://github.com/gravitational/teleport/blob/3dc51dd41112aa981b5d95a8f1842d2e859376c0/lib/events/slice.proto#L27

Reply:
I'm not sure we have specific guidelines for this. Here I was merely trying to be consistent with the field concerning max regular connections.